conf/routes

```diff
@@ -44,7 +44,11 @@ PUT            /api/stubs/:stubId/workingTitle             controllers.Api.putSt
 PUT            /api/stubs/:stubId/priority                 controllers.Api.putStubPriority(stubId: Long)
 PUT            /api/stubs/:stubId/trashed                  controllers.Api.putStubTrashed(stubId: Long)
 PUT            /api/stubs/:stubId/status                   controllers.Api.putStubStatus(stubId: Long)
-PUT            /api/stubs/:stubId/commissionedLength       controllers.Api.putStubCommissionedLength(stubId: Long)
+PUT            /api/stubs/:stubId/plannedPublicationId     controllers.Api.putStubPlannedPublicationId(stubId: Long)
+PUT            /api/stubs/:stubId/plannedBookId            controllers.Api.putStubPlannedBookId(stubId: Long)
+PUT            /api/stubs/:stubId/plannedBookSectionId     controllers.Api.putStubPlannedBookSectionId(stubId: Long)
+PUT            /api/stubs/:stubId/plannedNewspaperPageNumber          controllers.Api.putStubPlannedNewspaperPageNumber(stubId: Long)
+# PUT            /api/stubs/:stubId/plannedNewspaperPublicationDate     controllers.Api.putStubPlannedNewspaperPublicationDate(stubId: Long)
```

Comment:
Should we have a single endpoint for planned newspaper data?

The current `PUT` routes in this file appear to be for fields that are independent of each other and the data that constitute planned paper information is tightly coupled.

Having a single endpoint would allow for stronger input validation, I think, too.

Reply:
There aren't any `PUT` routes in this project which combine multiple fields. I don't think this warrants a second type of `PUT` endpoint, considering that the existing ones are so similar to each other in implementation.

Reply:
Also, if these three fields are so bound- then they should probably be modeled as a case clas..

Reply:
This probably should have been part of https://github.com/guardian/workflow-frontend/pull/226 can this model change be raised as a new card in trello?